app.js

```diff
@@ -11,26 +11,26 @@ const app = new App({
   logLevel: LogLevel.DEBUG,
 });
 
-/** Sample Function Listener */
-app.function('sample_function', async ({ client, inputs, fail }) => {
+/** Sample Step Listener */
```

Comment:
```suggestion
/** Sample Function Listener */
```

It's still a function listener. :)